docs/introduction/composition.md

```diff
@@ -599,16 +591,16 @@ the `spec.parameters.storageGB` field would immediately be propagated to the
 due to the relationship established between these two fields by the patches
 configured in the `example-azure` `Composition`.
 
-`kubectl describe` may be used to examine a composite resource. Note the
-`Synced` and `Ready` conditions below. The former indicates that Crossplane is
-successfully reconciling the composite resource by updating the composed
-resources. The latter indicates that all composed resources are also indicating
-that they are in condition `Ready`, and therefore the composite resource should
-be online and ready to use. More detail about the health and configuration of
-the composite resource can be determined by describing each composite resource.
-The kinds and names of each composed resource are exposed as "Resource Refs" -
-for example `kubectl describe mysqlserver example-zrpgr` will describe the
-detailed state of the composed Azure `MySQLServer`.
+`kubectl describe` may be used to examine a composite resource. Note the `Ready`
+condition below. It indicates that all composed resources are indicating they
+are 'ready' (according to the `readinessChecks` defined by the `Composition`),
```

Comment:
Maybe let them know you don't need to define any `readinessChecks` for managed resources? In fact, maybe we can omit `readinessChecks` here since it only applies to `ProviderConfig` if they use composition with XRM resources.

Reply:
I mostly wanted to work in the fact that custom readiness checks were a thing since they're not documented anywhere easy to find at the moment. I agree it's a bit awkward to mention them here though, so I've added them to the example Composition with a little comment explaining them.